src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2964,7 +2971,7 @@ namespace ThermalComfort {
 
         if (BeginDayFlag && !firstDaySet) {
             // Update the running average, reset the daily avg
-            runningAverageCEN = 0.2 * runningAverageCEN + 0.8 * avgDryBulbCEN;
+            runningAverageCEN = alpha * runningAverageCEN + (1.0 - alpha) * avgDryBulbCEN;
```

Comment:
T_rm = (1 - alpha) * T_od-1 + alpha * T_rm-1, according to the cen standard

Reply:
OK, so this won't make any difference because alpha was already fixed as a constant 0.8, but I like that there are fewer magic numbers here anyway.

Reply:
Not exactly, the original formula has incorrectly switched coefficients, where alpha (0.8) should be multiplied by the running mean temperature instead of the dry bulb. This bit is the biggest change in the algorithm in this PR I suppose.